doc/config/configuration_reference.rst

```diff
@@ -203,6 +203,15 @@ paths within containers.
 It is OK to perform this under ``/var`` or ``/opt`` or a novel path that your
 site maintains (e.g. ``/scratch``).
 
+.. _config-reference-insecureRegistries:
+
+insecureRegistries (array, OPTIONAL)
+----------------------------
```

Comment:
Could you extend the underline to the full length of the section title?
Sphinx will issue a warning when the underline is too short, and the testing scripts treat warnings as errors.

Apologies for the late notice, I only caught this detail when I ran the code through our internal pipeline.

Reply:
Fixed in [commit 5796f01](https://github.com/eth-cscs/sarus/pull/24/commits/5796f01eef37839bf3c4c783ef36b3f6c4d0e1ad)